SimpleWAWS/Code/CsmExtensions/CsmSubscriptionExtensions.cs

```diff
@@ -84,6 +145,31 @@ public static MakeSubscriptionFreeTrialResult MakeTrialSubscription(this Subscri
 
             return result;
         }
+        public static MakeSubscriptionFreeTrialResult MakeJenkinsTrialSubscription(this Subscription subscription)
```

Comment:
This also doesn't look that different from the other make trial sub function as far as I can see, does it?